### DISCUSSION TITLE
Spook lite version: repairs only

### DISCUSSION THREAD

**Discussion by erkr** [frenck/spook#738] [Ideas] [answered]
Of course I didn't install spook 🤭
The extra repairs checks are really time saving. I love them!!
I sanitise my configurations continuously, but still 10 repairs where found by Spook( only two false positives 🥳 )
BUT personally I don’t need the various extensions, nor the 62 extra entities introduced. 
Would it be feasible to install Spook in a ‘Lite’ flavour, offering the extra repairs part only?!
Thanks for this nice integration 🫡

**Comment by frenck** [comment, marked as answer]
> BUT personally I don’t need the various extensions, nor the 62 extra entities introduced.

You can disable them if you want too 🤷 

> Would it be feasible to install Spook in a ‘Lite’ flavour, offering the extra repairs part only?

Sorry, that won't happen.

../Frenck

**Comment by erkr** [comment]
You’re the boss ☺️
The entries I can disable, I know. But not the other extensions like the extra services

**Reply by frenck** [reply]
The extra services won't change a thing. They are not used by itself 🤷

**Reply by erkr** [reply]
Thanks, I disabled most of entries (except the repair related) and enjoy the improved repair support. Those are really useful 🥳
